flink-yarn/src/test/java/org/apache/flink/yarn/YarnApplicationFileUploaderTest.java

```diff
@@ -29,6 +29,7 @@
 import org.apache.hadoop.yarn.conf.YarnConfiguration;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.io.TempDir;
+import org.mockito.Mockito;
```

Comment:
Please don't use Mockito, as outlined in the Flink code style & quality guide https://flink.apache.org/contributing/code-style-and-quality-common.html#avoid-mockito---use-reusable-test-implementations

Reply:
Thanks for the comment, I've changed it to use a custom mocked class. I didn't create a new Java file for it as it's pretty specific to this test and unlikely to be re-used by other test classes